so-eclipse-plugin/src/com/sohelper/ui/QuestionPage.java

```diff
@@ -77,9 +78,15 @@ public void widgetSelected(SelectionEvent event) {
 				
 		Button votedUpCheckbox = new Button(optionsCont, SWT.CHECK);
 		Label onlyVotedUpLbl = new Label(optionsCont, SWT.NONE);
-		onlyVotedUpLbl.setText("Voted only (soon)");
-		votedUpCheckbox.setSelection(false); 
-		votedUpCheckbox.setEnabled(false);
+		onlyVotedUpLbl.setText("Voted only");
+		
+		votedUpCheckbox.addSelectionListener(new SelectionAdapter() {
+			@Override
```

Comment:
Not sure about this. I have tried but not able to do it. So as of now I am keeping the same.